quora-dark.user.css

```diff
@@ -149,9 +149,18 @@
     color: #ddd !important;
     background-color: #333 !important;
   }
+  .qu-bg--white, .qu-bg--white__ignore_dark_mode, .qu-bg--blue_ultralight {
+    background-color: #363636;
+  }
   .qu-bg--white, .qu-bg--white__ignore_dark_mode, .qu-bg--gray_ultralight {
     background-color: #282828;
   }
+  .q-platform--desktop .qu-hover--bg--blue_light:hover {
+    background-color: #444 !important;
```

Comment:
this color probably wants to be `#182030` rather than `#444444` since its a `--bg--blue_light`

Reply:
Hmm, I didn't even see the blue shade in the original light style on my external monitor. You're right but then maybe the not hovered version `--blue_ultralight` should have a blue shade too. `#182030` for unread and `#224466` for hovered looks nice. What should it be?

Reply:
that sounds good =)